<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SnoozeTimeSettingsBasic</class>
 <widget class="QDialog" name="SnoozeTimeSettingsBasic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="snoozeTimeAccept">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>170</y>
     <width>170</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color:#3B3B3B;
color:#BDBDBD;
border:none;
border-radius:3px;
}
QPushButton:pressed {
background-color:#606060;
color:#BDBDBD;
border:none;
border-radius:3px;
}
</string>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </widget>
  <widget class="QLabel" name="snoozeTimeUnit">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>100</y>
     <width>50</width>
     <height>14</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#616161</string>
   </property>
   <property name="text">
    <string>min</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="snoozeTimeTitle">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>16</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#BDBDBD</string>
   </property>
   <property name="text">
    <string>Snooze Time</string>
   </property>
  </widget>
  <widget class="QSlider" name="snoozeTimeSlider">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>125</y>
     <width>217</width>
     <height>32</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::handle:horizontal {
    background: #8f8f8f;
    width: 20px;
    border-radius: 7px;
}</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="snoozeTimeIncrement">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>72</y>
     <width>66</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color:#3B3B3B;
color:#BDBDBD;
border:none;
border-radius:3px;
}
QPushButton:pressed {
background-color:#606060;
color:#BDBDBD;
border:none;
border-radius:3px;
}
</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="snoozeTimeCancel">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>170</y>
     <width>170</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color:#3B3B3B;
color:#BDBDBD;
border:none;
border-radius:3px;
}
QPushButton:pressed {
background-color:#606060;
color:#BDBDBD;
border:none;
border-radius:3px;
}
</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="snoozeTimeDecrement">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>118</y>
     <width>66</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color:#3B3B3B;
color:#BDBDBD;
border:none;
border-radius:3px;
}
QPushButton:pressed {
background-color:#606060;
color:#BDBDBD;
border:none;
border-radius:3px;
}
</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="snoozeTimeValue">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>50</y>
     <width>50</width>
     <height>44</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#2196F3</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>349</width>
     <height>213</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(49, 49, 49);
border: 1px;
border-radius: 5px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>snoozeTimeAccept</zorder>
  <zorder>snoozeTimeUnit</zorder>
  <zorder>snoozeTimeTitle</zorder>
  <zorder>snoozeTimeSlider</zorder>
  <zorder>snoozeTimeIncrement</zorder>
  <zorder>snoozeTimeCancel</zorder>
  <zorder>snoozeTimeDecrement</zorder>
  <zorder>snoozeTimeValue</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
